frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -1,105 +1,141 @@
 # Contributing Data
 
-CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
+CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! Please see below for If you have single-cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
 
 ## Publishing Process
 
-The process for submission to the portal is:
-
-- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
-- We confirm that we will accept your data.
-- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
-- We upload to a private Collection where you can review.
-- You prepare revised data and send us your revised files, as needed.
-- We publish the data when you tell us to.
+1. To confirm we can support your submission, please [contact us] with a description of the data that you'd like to contribute
+2. We confirm that we will accept your data
+3. You send us files prepared according to the submission [requirements](#dataset-requirements)
+4. We upload to a private Collection where you can review
+5. The submission can be revised, as needed
+6. The data are made openly available when you are ready
 
 ## Dataset Requirements
 
 ### Data Eligibility
 
-CELLxGENE is focused on supporting the global community attempting to create references of human cells and tissues. As a result, there are a few kinds of datasets that we are **not** likely to accept at this time:
+CELLxGENE supports most single-cell RNA-seq and ATAC-seq data, but there are a few types of data that we are **not** likely to accept at this time:
```

Comment:
Should it simply be "tha tare not accepting at this time"? I realize that the "likely" qualifier is in the original text, but I'm unsure what is signifies.